### PR TITLE
https://gwrc.atlassian.net/browse/TEAMU-929 : Daily Aggregated Observations Multiple values for same day

### PR DESCRIPTION
**Pull request by swapnasam**
The .sql file will essentially remove daily duplicates from the core table.

Acceptance criteria
- View should not need the measurement_name column 
- Add the site name to the view, rather than site id
- Must only be one observation per site, per day

### CODE REVIEW

File: packages/Manager/src/main/resources/db/migration/V0040__observed_water_use_aggregated_daily.sql [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+        MIN(day_observed_at) OVER (PARTITION BY site_name) AS min_duplication_date
+    FROM
+        daily_calculation
+    GROUP BY
+        site_name, day_observed_at
+    HAVING
+        COUNT(DISTINCT measurement_name) > 1
+),
+
+PreviousMeasurement AS (
```
Comment (smozely) [inline packages/Manager/src/main/resources/db/migration/V0040__observed_water_use_aggregated_daily.sql:52]: This, and others ... keep the naming consistent i.e. `snake_case` which is fairly standard for Postgres.

File: packages/Manager/src/main/resources/db/migration/V0040__observed_water_use_aggregated_daily.sql [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+            WHEN d.day_observed_at IS NULL THEN 'No Duplication'
+            ELSE 'Duplication'
+        END AS duplication_status
+    FROM
+        daily_calculation dc
+    LEFT JOIN
+        duplication_dates d ON dc.site_name = d.site_name AND dc.day_observed_at = d.day_observed_at
+ 
+),
+NonDuplicateMeasurements AS (
```
Comment (smozely) [inline packages/Manager/src/main/resources/db/migration/V0040__observed_water_use_aggregated_daily.sql:68]: as above

File: packages/Manager/src/main/resources/db/migration/V0040__observed_water_use_aggregated_daily.sql [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+        daily_usage,
+        duplication_status,
+        CASE
+            WHEN duplication_status = 'No Duplication' THEN measurement_name
+            ELSE NULL
+        END AS non_duplicate_measurement
+    FROM
+        PreviousMeasurement
+),
+FilledMeasurements AS (
```
Comment (smozely) [inline packages/Manager/src/main/resources/db/migration/V0040__observed_water_use_aggregated_daily.sql:82]: As above

**Review by smozely — CHANGES_REQUESTED** [review]
I know we are talking later, but just first comments about keeping formatting consistent

File: packages/Manager/src/main/resources/db/migration/V0040__observed_water_use_aggregated_daily.sql (hunk 2 above)
Comment (swapnasam) [reply packages/Manager/src/main/resources/db/migration/V0040__observed_water_use_aggregated_daily.sql]: I have now changed. Thanks

**Review by smozely — APPROVED** [review]

**Pull request merged**